pandas/io/common.py

```diff
@@ -5,10 +5,24 @@
 import zipfile
 from contextlib import contextmanager, closing
 
-from pandas.compat import StringIO, string_types, BytesIO
+from pandas.compat import StringIO, BytesIO, string_types, text_type
 from pandas import compat
 
 
+try:
+    import pathlib
+    _PATHLIB_INSTALLED = True
+except ImportError:
+    _PATHLIB_INSTALLED = False
+
+
+try:
+    from py.path.local import LocalPath
```

Comment:
This is broken, you are skipping this test when it *should* work because its not imported correctly

```
In [1]: import py

In [2]: py.path.local.LocalPath
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-ac7907368c6c> in <module>()
----> 1 py.path.local.LocalPath

AttributeError: type object 'LocalPath' has no attribute 'LocalPath'

In [3]: py.path.local          
Out[3]: py._path.local.LocalPath
```